apps/bfd-model/bfd-model-codegen/pom.xml

```diff
@@ -49,7 +49,7 @@
 			<!-- Makes programmatically generating Java sources much simpler. -->
 			<groupId>com.squareup</groupId>
 			<artifactId>javapoet</artifactId>
-			<version>1.9.0</version>
+			<version>1.13.0</version>
```

Comment:
should use the new version property for java poet

Reply:
Fixed